pkg/logs/client/http/destination.go

```diff
@@ -322,6 +324,7 @@ func (d *Destination) unconditionalSend(payload *message.Payload) (err error) {
 
 	defer resp.Body.Close()
 	response, err := io.ReadAll(resp.Body)
+	log.Tracef("Log agent payload resolved with: %s", resp.Proto)
```

Comment:
I think we can still combine this with the above log line? At line `313` we already have the `resp` instance. 
Trace logs are already very noisy so i'd suggest merging them. (essentially keep the old log line you had, but use `resp.Proto` instead of `req.proto`.